### DISCUSSION TITLE
Welcome to rust-fitsio Discussions!

### DISCUSSION THREAD

**Discussion by simonrw** [simonrw/rust-fitsio#171] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. We are really interested to know how you are using this library - for what amazing astronomy projects are you using Rust?! :joy:

**Comment by cjordan** [comment]
I'm a postdoc at Curtin University developing CUDA-enabled calibration software for the MWA: https://github.com/MWATelescope/mwa_hyperdrive

I've re-written a lot of astro stuff in Rust, including some [coordinate transformations](https://github.com/MWATelescope/Marlu).  I personally think the investment in Rust has been absolutely worth it, so I'd like to see more people in astro writing Rust code.

**Comment by simonrw** [comment]
Hi all, I used to be a postdoc astrophysicist working in the UK on the [NGTS](http://ngtransits.org/) project. Since then I have moved into software engineering as a career, but still enjoy hearing about astro!

I think Rust has a great future in astronomy, particularly in data processing pipelines, observatory control, and other high performance analysis work.
